src/reducers/notifications.ts

```diff
@@ -163,23 +163,23 @@ function elipsis(str: string, n: number) {
  * A map of messages to show for each type of action.
  */
 const messages: {[key: string]: (proposalTitle: string | undefined, options: any) => string} = {
-  'GenesisProtocol.vote': (proposalTitle, {vote, proposalId}: Arc.VoteOptions) =>
```

Comment:
we probably want to replace all these types, they are useful and maybe necessary to have in some places